src/main/java/com/aws/greengrass/clientdevices/auth/metrics/handlers/GetClientDeviceAuthTokenMetricHandler.java

```diff
@@ -0,0 +1,46 @@
+/*
```

Comment:
Similarly, this isn't listening for `GetClientDeviceAuthTokenEvents`, it's listening to session creation events